front-end/src/search/SearchForm.js

```diff
@@ -1,41 +1,43 @@
-import React, { useState } from "react";
+import React from "react";
 import { listReservations } from "../utils/api";
+import ErrorAlert from "../layout/ErrorAlert";
 
 function SearchForm({
   setReservations,
-  reservations,
-  setSearch,
-  search,
+  setPhoneNumber,
+  phoneNumber,
   setShowList,
+  setSearchError,
+  searchError,
 }) {
   function handleChange(event) {
-    setSearch({
+    setPhoneNumber({
```

Comment:
Why are we setting the phone number here as a keyed object? arent we just accessing the value underneath directly anyways? can we just set this to the string value?